site/content/tutorial/12-actions/01-actions/app-a/App.svelte

```diff
@@ -35,15 +19,15 @@
 		top: calc(50% - var(--height) / 2);
 		border-radius: 4px;
 		background-color: #ff3e00;
-		cursor: move;
+		color: #fff;
+		text-align: center;
+		font-weight: bold;
 	}
 </style>
 
-<div class="box"
-	on:panstart={handlePanStart}
-	on:panmove={handlePanMove}
-	on:panend={handlePanEnd}
-	style="transform:
-		translate({$coords.x}px,{$coords.y}px)
-		rotate({$coords.x * 0.2}deg)"
-></div>
\ No newline at end of file
+<button on:click={() => (showModal = true)}>Show Modal</button>
+{#if showModal}
+	<div class="box" on:outclick={handleOutclick}>
```

Comment:
To me it's inconsistent that a lambda function is used to show the modal but a named function is used to close it. I suggest to make this an inline lambda function, too.